.travis.yml

```diff
@@ -2,7 +2,7 @@ language: scala
 
 # Only build non-pushes (so PRs, API requests & cron jobs) OR tags OR forks OR main branch builds
 # https://docs.travis-ci.com/user/conditional-builds-stages-jobs/
-if: type != push OR tag IS present OR repo != lagom/lagom OR branch IN (master, 1.5.x, 1.4.x)
+if: type != push OR tag IS present OR repo != lagom/lagom OR branch IN (master, 1.6.x, 1.5.x, 1.4.x)
```

Comment:
We can drop `1.4.x` too.

Reply:
https://github.com/lagom/lagom/commit/faae32e5f2e68618bc217b841f90bbb317d26813